xml/System.Net.Http/HttpClient.xml

```diff
@@ -28,6 +28,10 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+
+ > [!WARNING]
+ > <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. Instantiating an HttpClient class for every request will exhaust the number of sockets available under heavy loads. This will result in SocketException errors.
```

Comment:
Thanks @jzabroski. Unfortunately, this recommendation is not complete.
We need to warn users also about having long-existing `HttpClient` instances as that may lead to stale DNS usage. That can lead to non-functional connections.

The recommendation should be either to:
* Use `HttpClient` instances as statics, which are recycled on regular basis, or
* Use `HttpClientFactor` which works around this problem, or
* Use static instances on .NET Core while setting `System.Net.Http.SocketsHttpHandler.PooledConnectionLifetime`.